config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -484,7 +484,7 @@ presubmits:
         - --node-test-args=LocalStorageCapacityIsolation=true --kubelet-flags="--cgroups-per-qos=true --cgroup-root=/"
         - --node-tests=true
         - --provider=gce
```

Comment:
Can you please add a comment explaining why `NodeFeature:DevicePluginProbe` is skipped and potentially a TODO item how to get it back.